prashant/HTML/Assignment7/homepage.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Document</title>
+        <link href="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
+        <script src="//maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
+        <script src="//code.jquery.com/jquery-1.11.1.min.js"></script>
+        <link rel="stylesheet" href="homepage.css">
+        <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+    </head>
+    <body>
+        <header>
+            <div class="fixed-top">
+                <div class="header"></div>
+                <nav class="navbar navbar-expand-lg navbar-dark mx-background-top-linear">
+                    <div class="container">
+                        <a  href="http://scadonsak.com/7bF" style="float: left; font-size: 30px;"><ul class="navbar-nav ml-auto">JobPortal.COM</ul></a>
+                        <div class="collapse navbar-collapse" style="float: right; margin-left: 1000px; font-size: 18px; " id="navbarResponsive">
+                            <ul class="navbar-nav ml-auto">
+                                <a class="nav-link" href="homepage.html">Home</a>
+                                <a class="nav-link" href="Login.html">About</a>
+                                <a class="nav-link" href="#">CreateResume</a>
+                                <a class="nav-link" href="#">Blog</a>
+                                <a class="nav-link" href="Login.html">Login</a>
+                                <a class="nav-link" href="personaldetail.html">Signup  </a>
+                            </ul>
+                        </div>
+                    </div>
+                </nav>
+            </div>
+        </header>   
+        <h1 style="text-align: center; margin-top: 150px; font-size: 50px;">Find The Best Job For Your Future</h1>
+        <div class="form-group">
+            <div style="margin-left: 300px; margin-top: 150px; ">
+                <input type="text" placeholder="Search Job Here" class="form-control" id="usr" style="width: 800px; ; height: 50px; font-size: 20px; padding-left: 20px;">
+                <button type="button" class="btn btn-primary" style="height: 60px; width: 200px; margin-left: -10px; background-color: blue;" >Search</button>
+            </div>
+        </div>     
+    </body>
+</html>
```

Comment:
add extra line after `</html> `.that is at the end.

Reply:
done